deployments/rialto/Bridge.Dockerfile

```diff
@@ -38,8 +39,9 @@ WORKDIR /parity-bridges-common
 ARG BRIDGE_REPO=https://github.com/paritytech/parity-bridges-common
 RUN git clone $BRIDGE_REPO /parity-bridges-common && git checkout master
 
+# TODO: remove || true once PR is merged? Reason: if branch has new projects (rialto-bridge-node) and master has no this project yet (it has bridge-node), build fails
```

Comment:
We may leave this `|| true`, or not. For now, I've marked it as to-be-removed in the future.